server/blob/blob-export-file/src/test/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanismTest.java

```diff
@@ -191,6 +197,31 @@ void exportingBlobShouldCreateAFileWithExtensionWhenDeclaringExtension() {
             });
     }
 
+    @Test
+    void exportingBlobShouldCreateAFileWithPrefixWhenDeclaringPrefix() {
+        BlobId blobId = blobStore.save(BLOB_CONTENT).block();
+        String filePrefix = "bob@james.org";
```

Comment:
IMO the file prefix should mention the fact that it is a deleted message `deleted-message-bob@james.org-xxx.eml`